api/src/main/resources/notebooks.yaml

```diff
@@ -0,0 +1,362 @@
+swagger: '2.0'
```

Comment:
Add a comment indicating where this was copied from.

Reply:
Can do.